adsmp/models.py

```diff
@@ -5,7 +5,7 @@
 from adsputils import get_date
 from datetime import datetime
 from dateutil.tz import tzutc
-from sqlalchemy import Column, Integer, BigInteger, String, Text, TIMESTAMP, Boolean, DateTime
```

Comment:
Why was this not needed?

Reply:
I don't have a definitive answer but changing the model breaks the primary key functionality so it doesn't auto increment or otherwise populate the id field. The workers are still able to write `BIGINT` even with the field set as `Integer` in the model.

Reply:
I'm wondering if BigSerial should be used?  https://stackoverflow.com/questions/5998406/what-is-the-acceptable-method-for-generating-an-auto-incrementing-primary-key-in

Reply:
That was my thought as well, but `SQLAlchemy` doesn't have a built in method for it: [discussion](https://github.com/sqlalchemy/sqlalchemy/discussions/9790). It looks like we could build our own but it feels odd to me that it would be the right way to go about this. Setting the type explicitly to `BigSerial` in the upgrade still results in the model failing if `Integer`->`BigInteger`.

Reply:
It concerns me if someone adds a function that explicitly calls the model class to format a record and it fails.  I call models.py in ADSJournalsDB a lot.  That said, how long is adsmp going to exist in its current form?  It may be fine if this is a stopgap fix if we're transitioning away from adsmp as written in 6-12 mos.

Reply:
I'm inclined to go with what you have here.  I do not think we should implement continuum unless we have a clear use case for it in the current pipeline.  It may be something to consider for a new pipeline.

Reply:
Wild shot (because Googling has not given me a clear picture of whether it'll actually help), but have you tried a later version of SQLAlchemy? We have to stay on v1.X for compatibility with Python 2.7, but we're currently using SQLAlchemy v1.3.16 and the latest w/ that major release is v1.4.50.

Reply:
Also, related to the continuum discussion: once we've put this fire out, we should consider adding a cleaning script to this change log table so we're not maintaining so much history. This table has got to be giant and I doubt we're using it often enough to justify keeping so much history.